src/application.ts

```diff
@@ -2,24 +2,28 @@
 // Node module: loopback-next-hello-world
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
-import { Application } from '@loopback/core';
-import { AuthenticationComponent, AuthenticationBindings } from '@loopback/authentication';
-import { validateApiSpec } from '@loopback/testlab'
-import { MyAuthStrategyProvider } from './providers/auth-strategy';
-import { HelloWorldController } from './controllers/hello-world';
-import { MySequence } from './sequence';
-import { UserRepository } from './repositories/user.repository';
-import { Class, Repository, RepositoryMixin } from '@loopback/repository';
+import {Application} from '@loopback/core';
+import {
+  AuthenticationComponent,
+  AuthenticationBindings
+} from '@loopback/authentication';
+import {validateApiSpec} from '@loopback/testlab';
+import {MyAuthStrategyProvider} from './providers/auth-strategy';
+import {HelloWorldController} from './controllers/hello-world';
+import {MySequence} from './sequence';
+import {UserRepository} from './repositories/user.repository';
+import {Class, Repository, RepositoryMixin} from '@loopback/repository';
 
 export class HelloWorldApp extends RepositoryMixin(Application) {
   constructor() {
     super({
       components: [AuthenticationComponent],
       repositories: [UserRepository],
-      sequence: MySequence,
+      sequence: MySequence
```

Comment:
We are actually using trailing comma both in loopback and loopback-next. See e.g. https://github.com/strongloop/loopback-next/blob/3698d76825b27175e2533ef378db1ebb5e6952ca/.prettierrc#L5